perma_web/perma/views/user_management.py

```diff
@@ -985,6 +985,9 @@ def settings_organizations(request):
     if pending_registrar:
         messages.add_message(request, messages.INFO, "Thank you for requesting an account for your library. Perma.cc will review your request as soon as possible.")
 
+    organizations = request.user.organizations.all().order_by('registrar')
+    orgs_by_registrar = {registrar : [org for org in orgs] for registrar, orgs in itertools.groupby(organizations, lambda x: x.registrar)}
```

Comment:
The issue with using dict() as opposed to building the dict yourself using the dict and list comprehensions is not the key, which is the registrar object in both cases, but in the value. When using dict(), the value is an <itertools._grouper object>, which as far as I can tell, the Django templating language can't iterate through correctly.